src/Symfony/Bridge/PhpUnit/Legacy/ForwardCompatTestTraitForV5.php

```diff
@@ -210,4 +214,68 @@ public static function assertIsIterable($actual, $message = '')
     {
         static::assertInternalType('iterable', $actual, $message);
     }
+
+    /**
+     * @param string $exception
+     *
+     * @return void
+     */
+    public function expectException($exception)
```

Comment:
Shouldn't calling this reset `$this->forwardCompatExpectedExceptionMessage, $this->forwardCompatExpectedExceptionCode` in case you call it several times in different test cases?

Reply:
I wondered the same and we figure out this is not needed: a new instance is created by phpunit for every test case. That makes sense as that prevents leaks from one test to another.